charts/stable/pod-gateway-setter/templates/admissionregistration.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: {{ include "common.names.fullname" . | quote }}
+  labels:
+    {{- include "common.labels" . | nindent 4 }}
+  annotations:
+    cert-manager.io/inject-ca-from: "{{ .Release.Namespace }}/{{ include "pod-gateway-setter.servingCertificate" . }}"
+webhooks:
+- name: "pod-policy.example.com"
```

Comment:
Shouldn't this name be something related to pod-gateway-setter?

Reply:
you are right - replacing example name :-)